### PR TITLE
Use the SDK to handle audio only room composite requests with the default template

### PR DESCRIPTION
**Pull request by biglittlebigben**

### CODE REVIEW

**Review by dennwc — APPROVED** [review]
Not very familiar with the codebase, but it looks good so far :+1:

File: pkg/config/base.go [hunk 1]
```diff
@@ -36,12 +36,13 @@ type BaseConfig struct {
...
-	MaxUploadQueue       int            `yaml:"max_upload_queue"`       // maximum upload queue size, in minutes
-	DisallowLocalStorage bool           `yaml:"disallow_local_storage"` // require an upload config for all requests
+	Logging                      *logger.Config `yaml:"logging"`                          // logging config
+	TemplateBase                 string         `yaml:"template_base"`                    // custom template base url
+	ClusterID                    string         `yaml:"cluster_id"`                       // cluster this instance belongs to
+	EnableChromeSandbox          bool           `yaml:"enable_chrome_sandbox"`            // enable Chrome sandbox, requires extra docker configuration
+	MaxUploadQueue               int            `yaml:"max_upload_queue"`                 // maximum upload queue size, in minutes
+	DisallowLocalStorage         bool           `yaml:"disallow_local_storage"`           // require an upload config for all requests
+	EnableRoomCompositeSDKSource bool           `yaml:"enable_room_composite_sdk_source"` // attempt to render supported audio only room composite use cases using the SDK source instead of Chrome
```
Comment (frostbyte73) [inline pkg/config/base.go:45]: Would prefer this to be automatic, egress config is already pretty extensive

**Review by frostbyte73 — APPROVED** [review]

File: pkg/config/base.go (hunk 1 above)
Comment (biglittlebigben) [reply pkg/config/base.go:45]: Considering there is a not insignificant risk with rolling this out, I think we need a way to enable it on a per cluster basis and to disable it easily. Is there a better way to achieve this?

Comment (frostbyte73) [reply pkg/config/base.go:45]: that's fair, can always remove it in the future

**Review by davidzhao — COMMENTED** [review]
lg

one thing I didn't see (possibly I've missed it), how do we handle tracks that are published/unpublished after the pipeline had started? 

does `buildSDKInput` get called again to cycle through all audio tracks again?

**Comment by biglittlebigben** [comment]
It ends up using the same logic as participant egress to handle track updates. I don't quite recall the flow at this time.

**Pull request merged**